static/app/views/dashboards/widgetBuilder/components/visualize.tsx

```diff
@@ -103,15 +161,21 @@ function Visualize() {
           // For charts, we show aggregate parameter options for the y-axis as primary options.
           // For tables, we show all string tags and fields as primary options, as well
           // as aggregates that don't take parameters.
+          const columnFilterMethod = isChartWidget
+            ? datasetConfig.filterYAxisAggregateParams?.(
+                field,
+                state.displayType ?? DisplayType.LINE
+              )
+            : field.kind === FieldValueKind.FUNCTION
+              ? datasetConfig.filterAggregateParams
+              : datasetConfig.filterTableOptions;
```

Comment:
Tbh I'm not entirely sure what the difference between `filterAggregateParams` and `filterYAxisAggregateParams` is supposed to be. There's a chance we can consolidate them together in the future.